[IT] Add `Beatmap/Category` (#8684)
ppy/osu-wiki: wiki/Beatmap/Category/it.md

# Categorie di beatmap

Le [beatmaps](/wiki/Beatmap) caricate online sono suddivise in categorie in base alla loro recente attività , stato della [nominazione](/wiki/Beatmap_ranking_procedure#qualification) e popolarità.

## Categorie presenti

### Mappe abbandonate

> La sezione "Mappe abbandonate" è un luogo dove risiedono le beatmap abbandonate dai loro [hosts](/wiki/Beatmap/Beatmap_host). Queste mappe non hanno classifiche, ma possono ancora essere scaricate e giocate e continuano a [contribuire alle statistiche di gioco](#unranked-beatmaps-statistics). Utilizzano l'icona del punto interrogativo (![](/wiki/shared/status/graveyard.png)) nella [schermata di selezione dei brani](/wiki/Client/Interface#song-select).

Reviewer: The redirection to a section wasn't translated so that's why it throws an error there

```suggestion
La sezione "Mappe abbandonate" è un luogo dove risiedono le beatmap abbandonate dai loro [hosts](/wiki/Beatmap/Beatmap_host). Queste mappe non hanno classifiche, ma possono ancora essere scaricate e giocate e continuano a [contribuire alle statistiche di gioco](#statistiche-delle-mappe-non-classificate). Utilizzano l'icona del punto interrogativo (![](/wiki/shared/status/graveyard.png)) nella [schermata di selezione dei brani](/wiki/Client/Interface#song-select).
```

> Una beatmap viene spostata automaticamente nelle mappe abbandonate dalle [Beatmap in lavorazione e in attesa](#work-in-progress-and-pending) quando non viene aggiornata per 4 settimane. L'autore può far rivivere la beatmap (cioè riportarla [In attesa](#work-in-progress-and-pending)) aggiornandola tramite il [sistema di caricamento delle beatmap](/wiki/Beatmapping/Beatmap_submission). Una beatmap nelle abbandonate non conta per il [limite totale di caricamento](/wiki/osu!supporter#increased-limits) di un mappatore , ma per riportarla in vita è necessario uno slot libero in sospeso.

Reviewer: ```suggestion
Una beatmap viene spostata automaticamente nelle mappe abbandonate dalle [Beatmap in lavorazione e in attesa](#mappe-in-lavorazione-e-in-attesa) quando non viene aggiornata per 4 settimane. L'autore può far rivivere la beatmap (cioè riportarla [In attesa](#mappe-in-lavorazione-e-in-attesa)) aggiornandola tramite il [sistema di caricamento delle beatmap](/wiki/Beatmapping/Beatmap_submission). Una beatmap nelle abbandonate non conta per il [limite totale di caricamento](/wiki/osu!supporter#increased-limits) di un mappatore , ma per riportarla in vita è necessario uno slot libero in sospeso.
```

> Una beatmap viene spostata automaticamente nelle mappe abbandonate dalle [Beatmap in lavorazione e in attesa](#mappe-in-lavorazione-e-in-attesa) quando non viene aggiornata per 4 settimane. L'autore può far rivivere la beatmap (cioè riportarla [In attesa](#mappe-in-lavorazione-e-in-attesa)) aggiornandola tramite il [sistema di caricamento delle beatmap](/wiki/Beatmapping/Beatmap_submission). Una beatmap nelle abbandonate non conta per il [limite totale di caricamento](/wiki/osu!supporter#increased-limits) di un mappatore , ma per riportarla in vita è necessario uno slot libero in sospeso.

Reviewer: :) 

```suggestion
Una beatmap viene spostata automaticamente nelle mappe abbandonate dalle [Beatmap in lavorazione e in attesa](#mappe-in-lavorazione-e-in-attesa) quando non viene aggiornata per 4 settimane. L'autore può far rivivere la beatmap (cioè riportarla [In attesa](#mappe-in-lavorazione-e-in-attesa)) aggiornandola tramite il [sistema di caricamento delle beatmap](/wiki/Beatmapping/Beatmap_submission). Una beatmap nelle abbandonate non conta per il [limite totale di caricamento](/wiki/osu!supporter#increased-limits) di un mappatore , ma per riportarla in vita è necessario uno slot libero in sospeso.
```
```suggestion
Una beatmap viene spostata automaticamente nelle mappe abbandonate dalle [Beatmap in lavorazione e in attesa](#mappe-in-lavorazione-e-in-attesa) quando non viene aggiornata per 4 settimane. L'autore può far rivivere la beatmap (cioè riportarla [In attesa](#mappe-in-lavorazione-e-in-attesa)) aggiornandola tramite il [sistema di caricamento delle beatmap](/wiki/Beatmapping/Beatmap_submission). Una beatmap nelle abbandonate non conta per il [limite totale di caricamento](/wiki/osu!supporter#limiti-aumentati) di un mappatore , ma per riportarla in vita è necessario uno slot libero in sospeso.
```

### Mappe in lavorazione e in attesa

> Le mappe in lavorazione (comunemente abbreviate in *WIP*) e in attesa sono le due categorie in cui vengono inseriti i beatmap al momento dell'invio iniziale. I beatmap in queste categorie possono passare alla categoria [Qualificate](#qualificati) dalla [nomina](/wiki/Beatmap_ranking_procedure#nominations) da parte di almeno due [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators). Questo di solito include una grande quantità di feedback da parte dei [modder](/wiki/Modding/Modder) attraverso il [modding](/wiki/Modding). Le beatmap in lavorazione e in attesa non hanno classifiche, ma [contribuiscono alle statistiche di gioco](#unranked-beatmaps-statistics). Utilizzano l'icona del punto interrogativo (![](/wiki/shared/status/pending.png)) nella schermata di selezione dei brani.

Reviewer: ```suggestion
Le mappe in lavorazione (comunemente abbreviate in *WIP*) e in attesa sono le due categorie in cui vengono inseriti i beatmap al momento dell'invio iniziale. I beatmap in queste categorie possono passare alla categoria [Qualificate](#qualificate) dalla [nomina](/wiki/Beatmap_ranking_procedure#nominations) da parte di almeno due [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators). Questo di solito include una grande quantità di feedback da parte dei [modder](/wiki/Modding/Modder) attraverso il [modding](/wiki/Modding). Le beatmap in lavorazione e in attesa non hanno classifiche, ma [contribuiscono alle statistiche di gioco](#statistiche-delle-mappe-non-classificate). Utilizzano l'icona del punto interrogativo (![](/wiki/shared/status/pending.png)) nella schermata di selezione dei brani.
```

### Qualificate

> Quando la mappa è qualificata siamo nell'ultima fase della [procedura di classificazione delle beatmap](/wiki/Beatmap_ranking_procedure). Queste beatmap sono state nominate da almeno due [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) che si sono assicurati che la beatmap soddisfi gli standard dei [criteri di classificazione](/wiki/Ranking_Criteria). Le beatmap messe nella sezione Qualificate entrano in una [coda di classificazione](/wiki/Beatmap_ranking_procedure/Ranking_queue), e alla fine saranno [Classificate](#ranked) se non vengono [squalificate](/wiki/Beatmap_ranking_procedure#nomination-resets) dalla coda. Questo periodo serve come ultima fase di controllo per correggere eventuali difetti ed errori che potrebbero essere sfuggiti.

Reviewer: ```suggestion
Quando la mappa è qualificata siamo nell'ultima fase della [procedura di classificazione delle beatmap](/wiki/Beatmap_ranking_procedure). Queste beatmap sono state nominate da almeno due [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) che si sono assicurati che la beatmap soddisfi gli standard dei [criteri di classificazione](/wiki/Ranking_Criteria). Le beatmap messe nella sezione Qualificate entrano in una [coda di classificazione](/wiki/Beatmap_ranking_procedure/Ranking_queue), e alla fine saranno [Classificate](#classificate) se non vengono [squalificate](/wiki/Beatmap_ranking_procedure#nomination-resets) dalla coda. Questo periodo serve come ultima fase di controllo per correggere eventuali difetti ed errori che potrebbero essere sfuggiti.
```

## Storia

### Approvate

La categoria delle beatmap approvate è stata utilizzata tra il 2008 e il 2014 per ospitare le beatmap che superavano i requisiti di lunghezza e punteggio massimo previsti dalle vecchie versioni dei [criteri di classificazione](/wiki/Ranking_Criteria), oltre a rare beatmap che infrangevano altre regole dei criteri.

Le beatmap approvate non hanno influenzato la classifica per punteggio degli utenti fino a poco dopo il passaggio a quella per [punti performance](/wiki/Performance_points). In seguito a questo cambiamento, lo stato di "Approvato" è stato utilizzato esclusivamente per le beatmap con più di 6 minuti di [tempo di gioco](/wiki/Gameplay/Drain_time).

Le beatmap approvate utilizzano l'icona della spunta (![](/wiki/shared/status/approved.png)) nella schermata di selezione dei brani e funzionano come le beatmap classificate.

### Appearizione della sezione Qualificate

> La categoria [Qualificate](#qualified) è stata introdotta nell'agosto 2014 durante la [Ristrutturazione del BAT](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat) per dare maggiore visibilità alle beatmap appena classificate. Se una beatmap Qualificata fosse stata squalificata e spostata di nuovo in attesa, non avrebbe avuto alcun impatto sulle statistiche dei giocatori, perché la categoria stessa era stata progettata per dare punteggi transitori fin dall'inizio. Le aspettative preimpostate si sono dimostrate corrette e si è ridotto notevolmente il feedback negativo della comunità nei confronti delle declassificazioni.

Reviewer: ```suggestion
La categoria [Qualificate](#qualificate) è stata introdotta nell'agosto 2014 durante la [Ristrutturazione del BAT](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat) per dare maggiore visibilità alle beatmap appena classificate. Se una beatmap Qualificata fosse stata squalificata e spostata di nuovo in attesa, non avrebbe avuto alcun impatto sulle statistiche dei giocatori, perché la categoria stessa era stata progettata per dare punteggi transitori fin dall'inizio. Le aspettative preimpostate si sono dimostrate corrette e si è ridotto notevolmente il feedback negativo della comunità nei confronti delle declassificazioni.
```

### Statistiche delle mappe non classificate

> Dopo il [7 agosto 2020](https://osu.ppy.sh/home/changelog/stable40/20200807.3), le beatmap [Abbandonate](#graveyard) e [In lavorazione e in attesa](#work-in-progress-and-pending) hanno iniziato a contribuire al conteggio delle partite giocate degli utenti e al punteggio totale. Hanno anche ricevuto statistiche di gioco sulle loro pagine individuali (conteggio delle partite, percentuale di successo e punti di fallimento), cosa che prima veniva mostrata solo per le beatmap [Qualificate](#qualified) e [Classificate](#ranked).

Reviewer: ```suggestion
Dopo il [7 agosto 2020](https://osu.ppy.sh/home/changelog/stable40/20200807.3), le beatmap [Abbandonate](#mappe-abbandonate) e [In lavorazione e in attesa](#mappe-in-lavorazione-e-in-attesa) hanno iniziato a contribuire al conteggio delle partite giocate degli utenti e al punteggio totale. Hanno anche ricevuto statistiche di gioco sulle loro pagine individuali (conteggio delle partite, percentuale di successo e punti di fallimento), cosa che prima veniva mostrata solo per le beatmap [Qualificate](#qualificate) e [Classificate](#classificate).
```

# Categorie di beatmap

> Le [beatmaps](/wiki/Beatmap) caricate online sono suddivise in categorie in base alla loro recente attività , [nominazioni](/wiki/Beatmap_ranking_procedure#qualification) stato e popolarità.

Reviewer: It's nomination status not nomination, status ecc... so it should be translated to something like this or siminar to this

```suggestion
Le [beatmaps](/wiki/Beatmap) caricate online sono suddivise in categorie in base alla loro recente attività, stato della [nominazione](/wiki/Beatmap_ranking_procedure#qualification) e popolarità.
```

Le beatmap qualificate hanno classifiche, ma non vengono assegnati [punti performance](/wiki/Performance_points) e tutti i punteggi vengono cancellati quando escono dalla categoria Qualificate. Usano l'icona della spunta (![](/wiki/shared/status/qualified.png)) nella schermata di selezione dei brani.

### Classificate

Le beatmap classificate sono approvate come contenuto ufficiale, nel senso che hanno raggiunto o superato gli standard dei [criteri di classificazione](/wiki/Ranking_Criteria). Permettono ai giocatori di competere nelle classifiche e di guadagnare [punti performance](/wiki/Performance_points) grazie ai punteggi ottenuti. Una volta che una beatmap ha raggiunto lo stato di classificata, il suo stato non può essere più modificato (tranne in casi straordinari). Le beatmap classificate utilizzano l'icona del doppio chevron verso l'alto (![](/wiki/shared/status/ranked.png)) nella schermata di selezione dei brani.

### Amate

> *Per la storia della categoria Amate vedere: [Storia delle beatmap Amate](/wiki/Beatmap/Storia_degli_Amati)*.

Reviewer: Don't translate the urls, the website auto redirects to the correct language.

```suggestion
*Per la storia della categoria Amate vedere: [Storia delle beatmap Amate](/wiki/Beatmap/History_of_Loved)*.
```
